Cargo.toml

```diff
@@ -73,6 +75,7 @@ ivf = { version = "0.1", path = "ivf/", optional = true }
 v_frame = { version = "0.1", path = "v_frame/" }
 av-metrics = { version = "0.5.1", optional = true }
 rayon = "1.0"
+crossbeam = { version = "0.7", optional = true }
```

Comment:
Current crossbeam version is 0.7.3, not sure if going to latest upstream version is better

Reply:
It will pick the latest from 0.7 whichever it is :)